### DISCUSSION TITLE
Forcing dark mode

### DISCUSSION THREAD

**Discussion by thecheis** [rafaelmardojai/firefox-gnome-theme#297] [Q&A] [answered]
Hi everyone

Is possible to force the dark theme without using GNOME/Adwaita dark on entire desktop?
It could supposedly be done by modifying colors / dark.css, but I haven't got any results.
Have I done something wrong about it? Any ideas?

Is possible that this option will be incorporated in future releases?

Greetings.

**Comment by thecheis** [comment]
Didn't work. It just opened a dark window, but doesn't work on browser reboots. After this, now whenever I run Firefox it launches two windows.

I use Arch Linux, if is of any use or could influence.

**Comment by rafaelmardojai** [comment, marked as answer]
You should be able to force the dark theme by editing `theme/colors/dark.css`.

The CSS structure is like this:

```css
@media (prefers-color-scheme: dark) {
    :root {
        /* A bunch of CSS variables */
    }
}
```

You need to remove the media query. The resulting file should look like this:

```css
:root {
    /* A bunch of CSS variables */
}
```

**Comment by thecheis** [comment]
Yeah, that works! Thanks!
